src/main/java/uk/gov/hmcts/reform/cpo/service/mapper/CasePaymentOrderMapper.java

```diff
@@ -13,14 +13,15 @@
 
     CasePaymentOrderEntity toEntity(CasePaymentOrder casePaymentOrder);
 
-    @Mapping(target = "id", ignore = true)
     @Mapping(target = "createdTimestamp", ignore = true)
+    @Mapping(target = "historyExists", ignore = true)
     CasePaymentOrderEntity toEntity(CreateCasePaymentOrderRequest createCasePaymentOrderRequest,
                                                 String createdBy);
 
     CasePaymentOrder toDomainModel(CasePaymentOrderEntity casePaymentOrderEntity);
 
-    @Mapping(target = "createdTimestamp", expression = "java(java.time.LocalDateTime.now())")
+    @Mapping(target = "createdTimestamp", ignore = true)
```

Comment:
try removing these unmapped properties annotations 🤞

Reply:
Confirmed resolved in 506b03e73b0b823cf5b2faffa91f349c83b37a83